pyccl/tests/test_tkkssc.py

```diff
@@ -128,3 +130,119 @@ def test_tkkssc_linear_bias_raises():
     """Test that it raises if the profile is not NFW."""
     with pytest.raises(TypeError):
         ccl.halos.halomod_Tk3D_SSC_linear_bias(COSMO, HMC, prof=GNFW)
+
```

Comment:
This restores a test that was lost in the last few PRs, but which is quite useful (it actually uncovered the bug mentioned above)